EmailValidator/Validation/MultipleValidationWithAnd.php

```diff
@@ -62,7 +62,8 @@ public function isValid($email, EmailLexer $emailLexer)
         $errors = [];
         foreach ($this->validations as $validation) {
             $emailLexer->reset();
-            $result = $result && $validation->isValid($email, $emailLexer);
+            $validationResult = $validation->isValid($email, $emailLexer);
+            $result = $result && $validationResult;
```

Comment:
How about using `$result = ($result && $validationResult);` to make the code readability?.

Reply:
if that's general code style i can change it, but for me doesn't make it more readable.